cassandra-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/Main.java

```diff
@@ -105,6 +105,7 @@ private static int _main() throws UnknownHostException {
         final String    mesosMasterZkUrl        =                       Env.option("MESOS_ZK").or("zk://localhost:2181/mesos");
         final long      failoverTimeout         = Long.parseLong(       Env.option("CASSANDRA_FAILOVER_TIMEOUT_SECONDS").or(String.valueOf(Period.days(7).toStandardSeconds().getSeconds())));
         final String    mesosRole               =                       Env.option("CASSANDRA_FRAMEWORK_MESOS_ROLE").or("*");
+        final String    dataDirectory           =                       Env.option("CASSANDRA_DATA_DIRECTORY").or("/var/lib/cassandra");  // TODO: Temporary. Will be removed when MESOS-1554 is released
```

Comment:
Hm - would be better to have the env option `CASSANDRA_DATA_DIRECTORY` completely optional. Means, it should be `null` if not specified (effectively leaving the data files ephemeral, but that's ok - we can mention the fact in the readme).